app/lib/services/index.ts

```diff
@@ -1,7 +1,9 @@
 import * as connect from './connect';
 import * as restApi from './restApi';
+import { getServerInfo } from './getServerInfo';
 
 export const Services = {
 	...connect,
-	...restApi
+	...restApi,
+	getServerInfo
```

Comment:
I don't think this function is a "Service", it is a repo method that has an API call.

Reply:
@GleidsonDaniel I was afraid of doing so and making it harder to review.
It's not a new file and moving it could mess up git to the point it would think it's a new file.
Tried it anyway in hope of being smart enough to understand it was just moved.
Did it and it's a new file 🤡 
Some git nerds could say I'm the one to blame by not knowing it enough to predict it, but idk...